aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -227,12 +227,14 @@ open class AztecText : AppCompatEditText, TextWatcher, UnknownHtmlSpan.OnUnknown
     private var onAudioTappedListener: OnAudioTappedListener? = null
     private var onMediaDeletedListener: OnMediaDeletedListener? = null
     private var onVideoInfoRequestedListener: OnVideoInfoRequestedListener? = null
+    var onEnterListener: OnEnterListener? = null
```

Comment:
Thanks for the suggestions. didn't make it implemented in this way, since I was already thinking to AztecRN without putting much attentions to 3rd party host apps of Aztec.
Done.